cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/PathBuilderTest.java

```diff
@@ -70,10 +70,10 @@ public void testQueryStringList() {
         String path = new PathBuilder(CruiseControlEndpoints.REBALANCE)
                 .withParameter(CruiseControlParameters.JSON, "true")
                 .withParameter(CruiseControlParameters.DRY_RUN, "false")
-                .withParameter(CruiseControlParameters.VERBOSE, "true")
-                .withParameter(CruiseControlParameters.SKIP_HARD_GOAL_CHECK, "false")
                 .withParameter(CruiseControlParameters.EXCLUDED_TOPICS, "test-.*")
                 .withParameter(CruiseControlParameters.GOALS, GOALS)
+                .withParameter(CruiseControlParameters.VERBOSE, "true")
+                .withParameter(CruiseControlParameters.SKIP_HARD_GOAL_CHECK, "false")
```

Comment:
why moving these two lines?

Reply:
there were some changes I had to make in the `PathBuilder` class because we didn't had `verbose` present in the endpoint. That was the reason I had move the positions here